pages/login.js

```diff
@@ -40,32 +41,38 @@ export default function Home() {
 
   const signInWithGoogle = async () => {
     const user = await signInWithGooglePopup();
+    try {
+      const res = createUser();
+      console.log({ res });
+    } catch (error) {
+      console.log(error);
+    }
   };
 
-  const handleSubmit = async event => {
+  const handleSubmit = async (event) => {
     event.preventDefault();
 
     try {
       const { user } = await signInAuthUserWithEmailAndPassword(
         email,
         password
       );
-      getApi().then(api => api.get('/api/hello'))
+      getApi().then((api) => api.get("/api/hello"));
```

Comment:
We no longer need this api request to happen.
Let's remove the api call and as well as definition in `/api/hello.js`